.github/actions/build-workspace-docker/action.yml

```diff
@@ -0,0 +1,83 @@
+name: "build ugbio workspace docker"
+description: "general action to build and push ugbio workspace docker"
+
+inputs:
+  workspace-folder:
+    description: "workspace folder under src"
+    required: true
+  docker-repo:
+    description: "docker image repository name"
```

Comment:
what do you mean by "docker image repository name" ?

Reply:
the ecr/gcr repo name without the specific image tag, like:
make_examples, trimmer, sorter etc.